lib/inspect_addedfiles.c

```diff
@@ -110,9 +111,8 @@ static bool addedfiles_driver(struct rpminspect *ri, rpmfile_entry_t *file)
         return true;
     }
 
-    /*
-     * Security checks first (works for single build runs as well.
-     */
+    /* We will skip checks for ignored files */
+    ignore = ignore_rpmfile_entry(ri, NAME_ADDEDFILES, file);
```

Comment:
I generally don't in the functions called by foreach_peer_file() because foreach_peer_file() handles the assert on ri and if file is ever NULL, we would know before the program starts running inspections because I catch that during build fetching and file peering.